### ISSUE TITLE
Update to 2.0.4 from 2.0.0-alpha2 displays different behaviour for a TextView

### ISSUE THREAD

**Issue by Ali-Rezaei** [androidx/constraintlayout#148]
Here you can find my layout : https://github.com/Ali-Rezaei/TMDb-Paging/blob/master/app/src/main/res/layout/fragment_detail.xml

There is a AppCompatTextView in the above layout : 

```
<androidx.appcompat.widget.AppCompatTextView
            android:id="@+id/details_title"
            android:layout_width="0dp"
            android:layout_height="0dp"
            android:ellipsize="end"
            android:text="@{tmdbItem.name}"
            android:textAppearance="@style/TextAppearance.AppCompat.Title"
            android:transitionName="@string/view_name_header_title" />
```

When I have 2.0.0-alpha2 textView animated from bottom to top in the detail screen as shown in this video : https://drive.google.com/file/d/13c1W83DRq-vO3Efi3TRVdb0nsWHO8NSw/view?usp=sharing

I updated to latest stable version (2.0.4) and you can see different behaviour for TextView : https://drive.google.com/file/d/13elngfi6Hh0cbhSiLARbAOlkAxxxLPa6/view?usp=sharing

Here you can find layoutDescription : https://github.com/Ali-Rezaei/TMDb-Paging/blob/master/app/src/main/res/xml/scene_show_details.xml

Here is the start : 

```
<Constraint
            android:id="@id/details_title"
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            android:layout_marginStart="24dp"
            app:layout_constraintBottom_toTopOf="@id/details_appbar_background"
            app:layout_constraintEnd_toEndOf="@id/details_appbar_background"
            app:layout_constraintStart_toEndOf="@id/details_poster"
            app:layout_constraintTop_toBottomOf="@id/details_appbar_background" />
```

Here is the end : 

```
<ConstraintSet android:id="@+id/end">

<Constraint
            android:id="@id/details_title"
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            android:layout_marginEnd="48dp"
            android:layout_marginStart="64dp"
            android:elevation="@dimen/z_app_bar"
            app:layout_constraintBottom_toBottomOf="@id/details_appbar_background"
            app:layout_constraintEnd_toEndOf="@id/details_appbar_background"
            app:layout_constraintStart_toStartOf="@id/details_appbar_background"
            app:layout_constraintTop_toTopOf="@id/details_appbar_background" />
```

**Comment by KRXXX** [comment]
ok

**Comment by Ali-Rezaei** [comment]
Fixed with 2.1.0-rc01
